packages/formation-react/src/components/DropDownPanel/DropDownPanel.stories.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import DropDownPanel from './DropDownPanel';
+
+export default {
+  title: 'Library/DropDownPanel',
+  component: DropDownPanel,
+};
+
+const Template = (args) => {
+  const [isOpen, setOpen] = useState(args.isOpen);
+
+  return (
+    <div style={{ backgroundColor: '#112e51' }}>
+      <DropDownPanel
+        {...args}
+        isOpen={isOpen}
+        clickHandler={() => setOpen(!isOpen)}
```

Comment:
Overriding these props here because I couldn't figure out another way to demonstrate the opening and closing of the panel since the open and close state isn't managed internally.